### ISSUE TITLE
Update to 0.31.4 failed

### ISSUE THREAD

**Issue by oywino** [zachowj/node-red-contrib-home-assistant-websocket#406]
### Describe the bug

Upgrade gives the following err log:

```
-----------------------------------------------------------
2021-05-23T10:51:48.726Z Install : node-red-contrib-home-assistant-websocket 0.31.4

2021-05-23T10:51:46.717Z npm install --no-audit --no-update-notifier --no-fund --save --save-prefix=~ --production node-red-contrib-home-assistant-websocket@0.31.4
2021-05-23T10:51:56.991Z [err] npm
2021-05-23T10:51:56.991Z [err]  ERR! code EEXIST
2021-05-23T10:51:56.991Z [err] npm ERR! path /data/node_modules/.bin/mustache
2021-05-23T10:51:56.997Z [err] npm
2021-05-23T10:51:56.997Z [err]  ERR!
2021-05-23T10:51:56.997Z [err]  Refusing to delete /data/node_modules/.bin/mustache: is outside /data/node_modules/mustache and not a link
2021-05-23T10:51:56.997Z [err] npm
2021-05-23T10:51:56.997Z [err]  ERR!
2021-05-23T10:51:56.997Z [err]  File exists: /data/node_modules/.bin/mustache
2021-05-23T10:51:56.997Z [err] npm
2021-05-23T10:51:56.997Z [err]  ERR!
2021-05-23T10:51:56.997Z [err]  Remove the existing file and try again, or run npm
2021-05-23T10:51:56.997Z [err] npm ERR!
2021-05-23T10:51:56.997Z [err]  with --force to overwrite files recklessly.
2021-05-23T10:51:57.100Z [err] 
2021-05-23T10:51:57.100Z [err] npm ERR! A complete log of this run can be found in:
2021-05-23T10:51:57.100Z [err] npm ERR!     /data/.npm/_logs/2021-05-23T10_51_57_002Z-debug.log
2021-05-23T10:51:57.110Z rc=1

```
### To Reproduce

1. Go to 'main menu'
2. Click on 'Manage Palette'
3. Scroll down to 'update to 0.31.4'

### Expected behavior

I expected the update to go smoothly - just as always before

### Screenshots

![image](https://user-images.githubusercontent.com/18447119/119258168-df59f500-bbc8-11eb-9adc-3d128ef55151.png)

### Example Flow

```
export goes here
```

### Environment (please complete the following information):

- Node Red Version: [1.3.5]
- Node.js Version: [10.24.1]
- NR Home Assistant Plugin Version: [0.30.1]
- Is Node Red running in Docker: [yes]
- HA Core Docker Version: [2021.5.5]

### Other (please complete the following information):

- Have you searched previous issues for duplicates?: yes

**Comment by zachowj** [comment]
It's an npm error. Delete the file `/data/node_modules/.bin/mustache` and reinstalled it.

**Comment by oywino** [comment]
I did, and it works - but the list now shows the _mustache_ file is no longer residing in this folder (?) but instead is inside the container itself. If that's correct - does it imply that _mustache_ now slips my backup scheme which only includes external folders?

```
/data/node_modules/.bin $ ls                                                                                                                                                                                                                                                                                                                
total 84                                                                                                                                                                                                                                                                                                                                    
drwxrwxrwx    2 root     root          4096 May 23 13:27 ./                                                                                                                                                                                                                                                                                 
drwxrwxrwx  176 root     root          4096 May 23 13:27 ../                                                                                                                                                                                                                                                                                
-rw-rw-rw-    1 root     root           895 Oct 26  1985 flat                                                                                                                                                                                                                                                                               
-rw-rw-rw-    1 root     root           157 Oct 26  1985 mime                                                                                                                                                                                                                                                                               
-rw-rw-rw-    1 root     root          1376 Jan 27  2018 multicast-dns                                                                                                                                                                                                                                                                      
lrwxrwxrwx    1 node-red node-red        24 May 23 13:27 mustache -> ../mustache/bin/mustache*                                                                                                                                                                                                                                              
-rw-rw-rw-    1 root     root          3468 Oct 26  1985 mustache.bak                                                                                                                                                                                                                                                                       
-rw-rw-rw-    1 root     root          5738 Dec 21  2018 sshpk-conv                                                                                                                                                                                                                                                                         
-rw-rw-rw-    1 root     root          4011 Apr 22  2016 sshpk-sign                                                                                                                                                                                                                                                                         
-rw-rw-rw-    1 root     root          3553 Oct 12  2018 sshpk-verify                                                                                                                                                                                                                                                                       
-rw-rw-rw-    1 root     root          1579 Oct 26  1985 uuid                                                                                                                                                                                                                                                                               
/data/node_modules/.bin $ 
```

**Comment by zachowj** [comment]
There's no need to backup node_modules as they get installed every time a package does.

**Comment by oywino** [comment]
Ok, thank you 👍
I just made my backup the "easy way" and included the whole folder tree down from /data  😊

**Comment by oywino** [comment]
I was maybe a bit too quick.
Updating appeared to be successful after doing what you told me to do, but now the entity becomes unavailable in HA shortly after restart.  I have tried restarting both NR and HA several times. The behaviour is consistant.
This maybe the same problem as reported in #404

The error reported in NR was (on a couple of occasions);

```
Node-RED custom integration has been removed from Home Assistant it is needed for this node to function.

```

But I made no changes to either NR or HA, Just did the upgrade and then restarted NR and HA a couple of times.
There are no entries in the HA log at all.
